decidim-elections/app/controllers/decidim/votings/admin/voting_attachment_collections_controller.rb

```diff
@@ -16,6 +17,10 @@ def after_destroy_path
         def collection_for
           current_voting
         end
+
+        private
+
+        def tab_menu_name = :decidim_votings_attachments_menu
```

Comment:
I know it's out of the current change set, but I've just seen it. Let's make it consistent with the rest of modules. 
You'd need to change it in the other places of course (I've left the suggestion). 
```suggestion
        def tab_menu_name = :votings_admin_attachments_menu
```